File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/database/base/RealmIntMapper.java
package com.lattis.ellipse.data.database.base;

import androidx.annotation.NonNull;

public class RealmIntMapper extends AbstractRealmDataMapper<Integer,RealmInt> {

    public RealmIntMapper() { }

    @NonNull
    @Override
    public RealmInt mapIn(@NonNull Integer integer) {
        return new RealmInt(integer);
    }

    @NonNull
    @Override
    public Integer mapOut(@NonNull RealmInt realmInt) {
        return realmInt.getValue();
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/database/base/RealmIntMapper.kt
package com.lattis.data.database.base

class RealmIntMapper : AbstractRealmDataMapper<Int, RealmInt>() {

    override fun mapIn(integer: Int): RealmInt {
        return RealmInt(integer)
    }

    override fun mapOut(realmInt: RealmInt): Int {
        return realmInt.value
    }
}
